Analysis/Core/include/Analysis/PairCuts.h

```diff
@@ -29,22 +36,26 @@ class PairCuts
                   Rho,
                   ParticlesLastEntry };
 
+  void SetHistogramRegistry(HistogramRegistry* registry) { histogramRegistry = registry; }
+
   void SetPairCut(Particle particle, float cut)
   {
+    LOGF(info, "Enabled pair cut for %d with value %f", static_cast<int>(particle), cut);
     mCuts[particle] = cut;
-    if (mControlConvResoncances == nullptr) {
-      mControlConvResoncances = new TH2F("ControlConvResoncances", ";id;delta mass", 6, -0.5, 5.5, 500, -0.5, 0.5);
+    if (histogramRegistry != nullptr && histogramRegistry->contains("ControlConvResoncances") == false) {
```

Comment:
You may want to correct the typo in "ControlConvResoncances"

Reply:
Thanks. Done.